src/ripple/protocol/impl/LedgerFormats.cpp

```diff
@@ -261,6 +261,50 @@ LedgerFormats::LedgerFormats()
             {sfPreviousTxnLgrSeq,    soeREQUIRED}
         },
         commonFields);
+
+    add(jss::Bridge,
+        ltBRIDGE,
+        {
+            {sfAccount,                  soeREQUIRED},
+            {sfSignatureReward,          soeREQUIRED},
+            {sfMinAccountCreateAmount,   soeOPTIONAL},
+            {sfXChainBridge,             soeREQUIRED},
+            {sfXChainClaimID,            soeREQUIRED},
+            {sfXChainAccountCreateCount, soeREQUIRED},
+            {sfXChainAccountClaimCount,  soeREQUIRED},
+            {sfOwnerNode,                soeREQUIRED},
+            {sfPreviousTxnID,            soeREQUIRED},
+            {sfPreviousTxnLgrSeq,        soeREQUIRED}
+        },
+        commonFields);
+
+    add(jss::XChainOwnedClaimID,
+        ltXCHAIN_CLAIM_ID,
```

Comment:
The symbolic name should be updated to match the revised text name: `ltXCHAIN_OWNED_CLAIM_ID`

Reply:
Fixed in 6527ffd5a0 [fold] Rename ledger object ids https://github.com/XRPLF/rippled/pull/4292#discussion_r1110403681